core/chains/evm/config/v2/config_ocr2_test.go

```diff
@@ -0,0 +1,14 @@
+package v2_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/smartcontractkit/chainlink/v2/core/internal/cltest"
+)
+
+func TestOcr2Config(t *testing.T) {
+	evmOcrCfg := cltest.NewTestChainScopedConfig(t) //fallback.toml values
+	require.Equal(t, uint32(5300000), evmOcrCfg.EVM().OCR2().Automation().GasLimit())
```

Comment:
This is testing `ocr2Config` right? In that case, I think the idiomatic Go naming would be:
```suggestion
func Test_ocr2Config(t *testing.T) {
	evmOCRCfg := cltest.NewTestChainScopedConfig(t) //fallback.toml values
	require.Equal(t, uint32(5300000), evmOCRCfg.EVM().OCR2().Automation().GasLimit())
```

Reply:
Fixed in [`84f922f` (#9606)](https://github.com/smartcontractkit/chainlink/pull/9606/commits/84f922fd3885d4a8ff29121bba8cb0e5597a68ca)